source/MaterialXGenGlsl/Nodes/HeightFieldToNormal.cpp

```diff
@@ -0,0 +1,197 @@
+#include <MaterialXGenGlsl/Nodes/HeightFieldToNormal.h>
```

Comment:
For consistency I think the file should be named HeightToNormalGlsl.h/cpp. The convention used for the other nodes is [NodeName]Glsl.